src/systemctl/systemctl.c

```diff
@@ -2262,6 +2307,11 @@ static void output_jobs_list(const struct job_info* jobs, unsigned n, bool skipp
                        on, unit_len, e ? e : j->name, off,
                        type_len, j->type,
                        on, state_len, j->state, off);
+
+                if (arg_jobs_after)
+                        output_waiting_jobs(bus, j->id, "GetJobAfter", "\tA job waits for this job: ");
```

Comment:
This is rather verbose:
```
 JOB UNIT                                          TYPE  STATE  
8165 run-r047194e808954ed3a3c7257b5f4d7f61.service start waiting
        This job waits for a job: 8164 (foobar.device/start)
8164 foobar.device                                 start running
        A job waits for this job: 8165 (run-r047194e808954ed3a3c7257b5f4d7f61.service/start)

2 jobs listed.
```
Something like this would be sufficient:
``` 
JOB UNIT                                          TYPE  STATE  
8165 run-r047194e808954ed3a3c7257b5f4d7f61.service start waiting
        waiting for job 8164 (foobar.device/start)
8164 foobar.device                                 start running
        blocking job 8165 (run-r047194e808954ed3a3c7257b5f4d7f61.service/start)

2 jobs listed.
```
I think this would actually be easier to parse because "waiting" and "blocking" stand out more.